.github/workflows/sync-helm-readme.yml

```diff
@@ -0,0 +1,90 @@
+# Workflow to sync helm chart README.md when values.yml is updated
+name: Sync Helm Chart README.md
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, labeled]
```

Comment:
This seems like a lot of events for us to run this on

Reply:
The default is actually [opened, synchronize, reopened]. I've added [labeled] so that it can be run on label, although it does not need to be (the label is useful for testing and sanity checking).

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target

Reply:
Also worth noting, because of the specified `path` below these lines, it will _only_ run when the values.yaml file is updated, not on every single PR.

Reply:
https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow

Can we use this to run workflow manually?

Reply:
I've updated this, it required me to add an additional step to double check if it's running in a PR or not (if outside of a PR, it can create one by itself, otherwise update the existing PR in context).